run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+cd ..
+echo 'Running tests...'
+
+EXCLUDE_TESTS=HTTPDataStoreCSVBenchmarks,HTTPDataStoreBenchmarks,MicroBenchmarks,TDBDataStoreBenchmarks,FakeDataStoreBenchmarks,AverageMicroBenchmarks,SqliteDataStoreBenchmarks,DataStoreCSVBenchmarks,TDBDataStoreCSVBenchmarks,AFF4Benchmark
```

Comment:
Yes, I'm excluding them because they are just benchmark tests. They pass if included now. I don't know why, but they are not excluded anyway if the label specified is "small". I don't know why the label exclusion doesn't work for them. Would you like me to investigate further? Perhaps the label directive is ignored for some reason.